exalearn-ui/src/app/services/audio-cloud.service.ts

```diff
@@ -1,10 +1,18 @@
 import { Injectable } from '@angular/core';
-import { of } from 'rxjs';
+import { AngularFireStorage } from '@angular/fire/storage';
+import { of, Observable } from 'rxjs';
+import { finalize } from 'rxjs/operators';
 
 @Injectable({
 	providedIn: 'root'
 })
 export class AudioCloudService {
+	/* 
+		There is Aleksey Olshevsky code.
+		He uses it for his audition component.
+		I'll remove it when we will receive data from the server 
+	*/
+
```

Comment:
Good use of the comment!
Just write it a bit differently using ''TODO''. Change the last line into something like:
''TODO: remove the following lines after we start receiving data from the server''